tests/test_wcs.py

```diff
@@ -2303,6 +2303,51 @@ def test_gsfitswcs():
     assert_raises(TypeError, galsim.GSFitsWCS, file_name, header='dummy')
     assert_raises(ValueError, galsim.GSFitsWCS, _data=('invalid',0,0,0,0,0,0))
 
+@timer
+def test_inverseab_convergence():
```

Comment:
Need to add to ifmain block at the bottom.  I noticed a few other tests here lacked a `@timer` decoration too.

I wonder, in fact, if we should just automate the ifmain stuff.  The following snippet seems to work:
``` python
    testfns = [v for k, v in vars().items() if k[:5] == 'test_']
    for testfn in testfns:
        testfn()
```
We could probably do the timer wrapping there too.

Possible downsides include it's slightly harder to enable just a single test this way, since you have to remember its name instead of just commenting out n-1 `test_*` lines.

Reply:
Lately, I usually run single tests as e.g. `nosetests -s test_wcs.py:test_inverseab_convergence`, so that kind of automatic thing seems fine with me.  It is a common thing for us to forget.  :)